### ISSUE TITLE
[Feature Request?] Stutter Fix

### ISSUE THREAD

**Issue by SpoogLord** [jshepler/jshepler.ngu.mods#3]
Hey, idk if this is from using your mod or if it happens in base NGU but if I leave the game open for about a day I start getting these weird stutters, its a consistent short stutter once every second, if needed I can submit footage. (you'll have to give me about 20 hours lol)

But for the actual request, if possible could you update the mod to fix that? 

Alternatively just adding a quick option to reload the game would work as well, maybe shift click "Save Game" to make a save, close the game, relaunch the game, then load the save? Though, I don't know if that's even possible without the mod being an external script.
or hear me out, some button to launch an external script that saves, kills, relaunches, and loads the game lol

If you aren't interested or whatever its not a huge deal, just thought I'd ask

**Comment by jshepler** [comment]
No clue what that is. It happens to me, but not consistently. Sometimes it clears itself up without needed to restart the game. Sometimes not. You're the first person to mention it to me, so I thought it was just something on my computer.

Restarting the game isn't that big a deal (F5, close, launch, F6) so I've never worried about it.

I'll leave this open though, Maybe someone who knows what's going on will chime in and if it's something I can fix, I will.

**Comment by baraklevy20** [comment]
For full disclosure, I use Cheat Engine to speed the game up (x60), but I face a similar stuttering issue after about 24 hours (in-game time). When it starts, the timer goes up by intervals of 20 seconds each frame (as opposed to 1 second each frame).

What's weird is that, while closing and reopening the game fixes it, navigating to the quests screen also fixes it. If I stay on that screen for the entire rebirth, there's no stuttering.
I've tried to use Dnspy to understand what's so special about this UI, but couldn't figure out the reason, unfortunately.

If this is still an issue when playing normally, could you try to see if navigating to the quests UI fixes it?

**Comment by jshepler** [comment]
> If this is still an issue when playing normally, could you try to see if navigating to the quests UI fixes it?

Yeah, I'll try that. Thanks.

**Comment by SpoogLord** [comment]
I had forgotten about this problem as it hasn't occurred to me in a long time, but I opened this before I unlocked quests.. He may be onto something

**Comment by jshepler** [comment]
Yeah, it works. At least I have some kind of direction to look in to see if I can figure out wtf is going on.

**Comment by jshepler** [comment]
I believe I found the issue. Fix will be in next update.

It's the mod that adds the quest item drop chance to the questing menu. It appends it to the end of the existing text, but when that menu isn't active, that existing text doesn't get refreshed, so the mod was continually appending another line.

<img width="330" height="277" alt="Image" src="https://github.com/user-attachments/assets/7b46a660-d331-4942-bf71-278fe55a62bd" />

Eventually, the text is so long that when another line is added, it takes too long to update and the frame is delayed. The update is done every 1 second and lines up perfectly with the stutter, so I'm pretty sure this is the problem.

**Comment by baraklevy20** [comment]
Oh wow, nice find. Thanks for taking care of this!

**Comment by jshepler** [comment]
You're the man for noticing that it cleared when going to the questing menu.

Thanks for sharing that observation. I wouldn't have found this otherwise.
